src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -215,12 +219,30 @@ const AccountTransfer = (() => {
                         return;
                     }
                     withdrawal_limit = +response_limits.get_limits.remainder;
+
                     if (withdrawal_limit < +min_amount) {
                         getElementById(messages.limit).setVisibility(1);
                         getElementById(messages.parent).setVisibility(1);
                         return;
                     }
-                    getElementById('range_hint').textContent = `${localize('Min')}: ${min_amount} ${localize('Max')}: ${client_balance <= withdrawal_limit ? localize('Current balance') : localize('Withdrawal limit')}`;
+                    max_amount = Currency.getMaxTransfer(
+                        Client.get('currency')
+                    );
+                    transferable_amount = max_amount ?
+                        Math.min(max_amount, withdrawal_limit, client_balance) :
+                        Math.min(withdrawal_limit, client_balance);
```

Comment:
regarding having a fallback value, I had a discussion with Farzaneh and she said that if API does not return rates to us we should have the account transfer form disabled and should show a message that rates are not available at this time and to try again later. can you check with her on that?

Reply:
We tested this together. right now, the only way to reproduce it is to stop exchange feed. in which it causes `get_limits` API to throw an `InternalServerError` and we don't show the form at all. So I thought, what if this doesn't happen in production? if `get_limits` API does not throw an internal server error, then the form will break. That was the reason I added the fallback value just to be on the safe side.

Reply:
sure, let's go with this then